apps/fluent-tester/src/testPages.ts

```diff
@@ -78,13 +78,13 @@ export const tests: TestDescription[] = [
     platforms: ['android', 'macos', 'win32'],
   },
   {
-    name: 'Checkbox',
+    name: 'Checkbox Pre-V1',
     component: CheckboxTest,
     testPage: HOMEPAGE_CHECKBOX_BUTTON,
     platforms: ['android', 'ios', 'macos', 'win32', 'windows'],
   },
   {
-    name: 'Checkbox (Experimental)',
+    name: 'Checkbox V1',
     component: ExperimentalCheckboxTest,
     testPage: HOMEPAGE_CHECKBOX_EXPERIMENTAL_BUTTON,
     platforms: ['android', 'ios', 'macos', 'win32'], // 'windows': GH#935: Temporarily disabling while SVGs don't work in windows
```

Comment:
Isn't the first one the V1 Checkbox page, while the second one is the experimental one (native module on macOS, and.. redirect to V1 elsewhere) ?

Reply:
No. The first checkbox page uses the deprecated checkbox. It does look like the second one tests the checkbox from experimental, which would use the native module on macOS and V1 on win32.

Reply:
Oh, thanks for the clarification. That's simpler than I thought :)

Reply:
@ejlayne for the future, remind me to ask you about moving the macOS native wrapper components to their own packages / a separate component in the same package. I think this is something we agreed to do in the past but didn't follow through with and is left as a hanging fruit.